session.go

```diff
@@ -475,6 +475,21 @@ func (s *Session) SetTrace(trace Tracer) {
 	s.mu.Unlock()
 }
 
+var selectWildcardNextWarning time.Time
+var mu sync.Mutex
+
+func warnSelectWildcardStartPerformanceIssue() {
+	mu.Lock()
```

Comment:
Don't do lock, locks are expensive, use `atomic.StoreInt64` and `time.Now().Unix()` instead, also take into account that we don't have to be precice here, so construction like would work just fine:
```
now := time.Now().Unix()
if atomic.LoadInt64(selectWildcardNextWarning) < now {
    atomic.StoreInt64(now + 60);
   ....
} 
```